src/Microsoft.Crank.Jobs.HttpClient/Program.cs

```diff
@@ -261,8 +311,9 @@ public static async Task<WorkerResult> DoWorkAsync()
                         counters[status / 100 - 1]++;
                     }
                 }
-                catch
+                catch (Exception e)
                 {
+                    Console.WriteLine(e);
```

Comment:
Remove now to prevent flooding the log when it goes wrong? Or maybe only on `socketErrors == 0`?